### PR TITLE
junit 5 assignment

### PR DESCRIPTION
**Pull request by SahilBabbar12**

### CODE REVIEW

File: JUnit5/src/test/java/com/knoldus/PersonTest.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    void getNameTest1() {
+        person.setName("Sahil Babbar");
+        Assertions.assertEquals("Sahil Babbar", person.getName());
+    }
+
+
+    // Test to set and getName with null value.
+    @Test
+    void getNameTest2() {
+        person.setName(null);
```
Comment (anup0060) [inline JUnit5/src/test/java/com/knoldus/PersonTest.java:31]: poor naming 

testGetName_whenNameNull

File: JUnit5/src/test/java/com/knoldus/PersonTest.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    // Test to  set and getAge with not-null value.
+    @Test
+    void getAgeTest1() {
+        person.setAge(20);
+        Assertions.assertEquals(20, person.getAge());
+    }
+
+    // Test to  set and getAge with null value.
+    @Test
+
```
Comment (anup0060) [inline JUnit5/src/test/java/com/knoldus/PersonTest.java]: why line gap

File: JUnit5/src/test/java/com/knoldus/PersonTest.java [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    }
+
+    // Test to get and setAge with negative integer
+    @Test
+    void setAgeTestForNegativeNumber() {
+        Assertions.assertThrows(IllegalArgumentException.class, () -> person.setAge(-11));
+    }
+
+    @Test
+    public void setAgeTest_ifAgeIsZero() {
```
Comment (anup0060) [inline JUnit5/src/test/java/com/knoldus/PersonTest.java]: use words - should, given, when in suffix , instead of if.

**Review by anup0060 — CHANGES_REQUESTED** [review]

**Review by SahilBabbar12 — COMMENTED** [review]
I have made the changes suggested by you. please review it again.

**Review by anup0060 — APPROVED** [review]

**Pull request merged**